packages/mobx/__tests__/v5/base/object-api.js

```diff
@@ -69,7 +69,10 @@ test("object - set, remove, values are reactive", () => {
     expect(snapshots).toEqual([[3], [3, 4], [5, 4], [5]])
 })
 
-test("object with symbol keys - set, remove, values are reactive", () => {
+// 21.12.2020 @urugator:
+// Since https://github.com/mobxjs/mobx/pull/2641
+// values() works like Object.values - symbols are not part of the result
+test.skip("object with symbol keys - set, remove, values are reactive", () => {
```

Comment:
Hm, actually I've bumped into an issue here ... 
`keysAtom.reportChanged` is called on any key addition/removal, so it over-reports on `values/keys/entries` when you add/remove non-enumerable or symbolic prop.
I can fix it by reporting only when the prop is enumerable and non-symbolic. The problem is that `ownKeys` won't report for these non-enumerable or symbolic prop.
I can't even solve this by having two atoms (`keysAtom`+`ownKeysAtom`), because `Object.keys(dynamicObject)` uses `ownKeys` trap, so it would subscribe for `ownKeysAtom` and over-report anyway.
So I can either over-report in `values/keys/entries` or under-report on `ownKeys`. I think the second variant is more practical, but it can actually break the app (cause staleness), while the first variant has "only" perf implications.
Any ideas?

Reply:
At first impression: I think over-reporting should be fine, as we have a recommendation somewhere anyway to either use an object purely as collection, (in which case we care about everything added and storing thing in additional 'non iterable' members shouldn't be really done), or as something with a static shape, in which case we ideally don't iterate anyway.

Reply:
Over-reporting it is - commented code, updated tests